source/views/building-hours/stateful-list.js

```diff
@@ -27,7 +29,31 @@ const groupBuildings = (buildings: BuildingType[]) => {
   return toPairs(grouped).map(([key, value]) => ({title: key, data: value}))
 }
 
-type Props = TopLevelViewPropsType
+const loadAllBuildings = (
```

Comment:
I'd like to see this logic incorporated into groupBuildings itself, if you don't mind. I can show you how if you'd like.